sig-data-science/charter.md

```diff
@@ -6,11 +6,11 @@ This charter adheres to the conventions described in the [Operate First Governan
 
 The Data Science SIG is responsible for all data science and machine learning related initiatives within the Operate First community. However, this SIG's primary focus is to serve the following 3 main objectives:
 
-* Use Operate First as the go-to platform for incubating Open Source cloud native data science projects and defining best practices for developing and operating such Open Source cloud native data science projects.
+* Use Operate First as the go-to platform for incubating Open Source cloud Native Data Science projects and defining best practices for developing and operating such Open Source Cloud Native Data Science projects.
```

Comment:
```suggestion
* Use Operate First as the go-to platform for incubating Open Source Cloud Native Data Science projects and defining best practices for developing and operating such Open Source Cloud Native Data Science projects.
```

Reply:
The editor in me loves the edit workflow here for such language changes and discussions. Like here, it gives us a chance to consider the changes together as it happens to be a chance to capture some reasoning and practice for the whole project.

So I'm curious about the context around these edits? In particular, what is the thinking around capitalizing:

1. "Cloud Native"
2. "Data Science"

Such as, a style guide that suggests those?

I've seen DS capitalized often, have wondered why, and had it in my mental list of glossary terms we need to standardize use for. Cloud native is similar, to me -- what's the common industry usage?

The tendency in English is to not capitalize nouns unless they are proper names, and in the marketspeak dialect of English we have gray lines for sure. I believe Red Hat style guides tend toward only capitalizing product names, for example. But I haven't looked for the treatment of data science yet. Is this a norm in the industry or academia?

Only very recently I've begun capitalizing Open Source as part of a practice to differentiate the special Opens like Open Science or any Open Work from open in name but not really in fact.

Word nerd time, let's go!

What do you think and can educate my ignorance on?

Reply:
> Thanks for the feedback @quaid . I agree the capitalization of these things throughout industry writing can be a little fuzzy. And when the rules are not clear I try to at least be consistent, with an arguably arbitrary decision, throughout a document.

+100 I would rather be consistently incorrect with a usage like that, because inconsistency causes doubt in the reader.

> As far the capitalization here, I was just trying to highlight the fact that I'm referring a single concept "Open Source Cloud Native Data Science". However, that might be confusing or unnecessary. Happy to change it to "open source cloud native data science". Lower case is probably more grammatically correct anyways.

Ah, I see what you were looking to do, this is where some hyphens can help the reader connect the proper words together so they aren't wondering what "native data" is.

How about this (with my recent preference for capitalizing Open Source included):
```suggestion
* Use Operate First as the go-to platform for incubating Open Source cloud-native data science projects and defining best practices for developing and operating such Open Source cloud-native data science efforts.
```

Swapping in "efforts" so the "project" word isn't doing all the heavy lifting.